src/features/deposit/services/depositService.ts

```diff
@@ -1,12 +1,16 @@
 import type { BlockchainEnum } from "src/types"
+import { settings } from "../../../config/settings"
 import {
   getDepositAddress,
   getSupportedTokens,
 } from "../../../services/poaBridgeClient"
 import type { Transaction } from "../../../types/deposit"
+import { getNearNep141StorageBalance } from "../../machines/getBalanceMachine"
 import { getNearTxSuccessValue } from "../../machines/getTxMachine"
 export const FT_MAX_GAS_TRANSACTION = `300${"0".repeat(12)}` // 300 TGAS
+export const FT_REDUCED_GAS_TRANSACTION = `270${"0".repeat(12)}` // 270 TGAS (for storage deposit)
 export const FT_DEPOSIT_GAS = `30${"0".repeat(12)}` // 30 TGAS
+export const FT_MINIMUM_STORAGE_BALANCE = 1250000000000000000000n
```

Comment:
It differs from token to token, so it's better to get this info from chain.

Reply:
Indeed, I'll open a separate PR for this.